### ISSUE TITLE
distilbert-base-uncased

### ISSUE THREAD

**Issue by anitaj-j** [huggingface/transformers#4239]
# 🐛 Bug/Question

## Information
 am trying to execute :

**import ktrain

from ktrain import text

MODEL_NAME='distilbert-base-uncased'

t=text.Transformer(MODEL_NAME, maxlen=500, classes=np.unique(y_train))**

I get the following error:

OSError: Model name 'distilbert-base-uncased' was not found in tokenizers model name list (distilbert-base-uncased, distilbert-base-uncased-distilled-squad, distilbert-base-cased, distilbert-base-cased-distilled-squad, distilbert-base-german-cased, distilbert-base-multilingual-cased). We assumed 'distilbert-base-uncased' was a path, a model identifier, or url to a directory containing vocabulary files named ['vocab.txt'] but couldn't find such vocabulary files at this path or url.

ktrain version 0.14.6 transformers version 2.8.0

Libraries was installed using pip install. I am following the tutorial : https://towardsdatascience.com/text-classification-with-hugging-face-transformers-in-tensorflow-2-without-tears-ee50e4f3e7ed 

Any help would be appreciated.

**Comment by LuCeHe** [comment]
hi! did you find a solution? I got the same problem!

**Comment by Gnomic20** [comment]
I have the same problem with a freshly installed version. Any ideas?

**Comment by amaiya** [comment]
This can happen if there is a network or firewall issue that prevents the `transformers` library from downloading the required files.   To replicate on a laptop, you can delete the cache (e.g., `~/.cache/torch/transformers`), turn off wifi, and run this:
```python
from transformers import *
tokenizer = DistilBertTokenizer.from_pretrained('distilbert-base-uncased')
```
If following the above steps, you will see the exact same error as posted above (i.e., `OSError: Model name 'distilbert-base-uncased' was not found in tokenizers model name list `).

As a workaround for machines with download issues, you can download the tokenizer files (e.g., `vocab.txt`) and model files manually using machine with access, copy folder to machine with firewall/network issues, and then point `transformers` to the folder containing the downloaded files (e.g., `DistilBertTokenizer.from_pretrained('path/to/folder)`).

See [this ktrain FAQ entry](https://github.com/amaiya/ktrain/blob/master/FAQ.md#how-do-i-use-ktrain-without-an-internet-connection) for more information on workarounds/solutions on using `transformers` without internet or behind firewall.  While the entry focuses on using `transformers` from within the  *ktrain* library, most information applies generally to `transformers`.

**Comment by ayushidalmia** [comment]
This is resolved if you ensure you use transformers==3.4.0
